web/src/components/organisms/atoms/Navbar.jsx

```diff
@@ -0,0 +1,21 @@
+import Link from 'next/link';
+import { HiLogout } from "react-icons/hi";
+
+const Navbar = () => {
+    return (
+<nav className="bg-gray-50 dark:bg-gray-800 p-2 text-center">
+      <div className="container mx-auto flex justify-between items-center">
+        <Link href="/" className="text-black font-bold justify-center">
```

Comment:
- Remove unnecessary class
-  remove color, let's use default
```suggestion
        <Link href="/" className="font-bold">
```